CHANGES.rst

```diff
@@ -1,6 +1,30 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool help documentation by grouping of commands and
+  usage examples.
+- Renames ``workflows`` to ``list`` and ``list`` to ``ls``.
+- Introduces ``du`` command for checking workspace disk usage.
+- Introduces ``mv`` command for moving files within workspace.
+- Introduces ``diff`` command for comparing two workflows.
+- Introduces ``rm`` command for removing files within workspace.
+- Introduces ``stop`` command for stopping a running workflow.
+- Introduces ``delete`` command for deleting a workflow.
+- Introduces ``run`` command which parses ``reana.yaml``, creates a workflow,
+  uploads its assets and starts its execution.
+- Introduces commands for opening, listing and closing interactive sessions.
+- Introduces ``--parameter`` flag for overriding workflow input parameters.
+- Introduces ``--option`` flag for passing operational options to ``start`` or
+  ``run`` commands. E.g. ``CACHE=off``.
+- Introduces ``--format`` flag for ``ls`` command.
+- Introduces ``--json`` flag for ``logs`` command.
+- Introduces new api function ``create_workflow_from_json``, which allows to
+  create a workflow from json specification.
+- Introduces new field ``resources`` in reana.yaml.
```

Comment:
We can highlight `reana.yaml`, explain what resources mean (it is kind [CWL Resource Requirements](https://www.commonwl.org/v1.0/CommandLineTool.html#ResourceRequirement) but we are having [only CVMFS mounts](https://reana.readthedocs.io/en/latest/userguide.html#declare-necessary-resources)) and add an example in the same fashion as in https://github.com/reanahub/reana-client/pull/282/files#r277598930